scripts/email_fluidx_sample_receipt.py

```diff
@@ -34,33 +30,19 @@ def _run(self):
 Kind regards,
 ClarityX'''
 
-        # fill in message with parameters
-        msg2 = msg2.format(
-            link='https://' + platform.node() + '/clarity/queue/752',
-            project=self.projects[0].name,
-        )
-        subject2 = ', '.join([p.name for p in self.projects]) + ': Parse Manifest Required (FluidX)'
+        msg2 = msg2.format(link='https://' + platform.node() + '/clarity/queue/752', project=self.projects[0].name)
```

Comment:
There was a request to change this in #42, but it doesn't seem to have happened. Should we do something about this here?

Reply:
I discussed this with Dan and It should be left as is for now